changelog.R

```diff
@@ -266,8 +278,7 @@ chglog <- tribble(
 )
 
 
-output$changelog <- renderHtmlTableWidget({ 
-  chglog %>% 
-    addHtmlTableStyle(align="l") %>% 
-    htmlTableWidget(rnames=FALSE) 
-})
+output$changelog <- renderDataTable({
+  chglog2 <- chglog
```

Comment:
Again, this works, but I am not sure why we're creating a dataframe named chglog2? It also works to just have "chglog" here